src/styles/main/_layout.scss

```diff
@@ -249,6 +245,6 @@ header {
   }
 
   .selectize-dropdown, .selectize-input, .selectize-input input {
-    font-size: 0.9em;
+    font-size: 0.75em;
```

Comment:
Does this graphically change Seattle's version at normal size?

Reply:
if we don't need to, then let's not give the existing codebase any grief. We can tune this for ourselves in the `bitters-springfield` overrides.